src/trustRegion.jl

```diff
@@ -25,6 +25,20 @@ states as `RadiusUpdateSchemes.T`. Simply put the desired scheme as follows:
     """
     Simple
 
+    """
+    `RadiusUpdateSchemes.NLsolve`
+
+    The same updating scheme as in NLsolve's (https://github.com/JuliaNLSolvers/NLsolve.jl) trust region dogleg implementation. 
+    """
+    NLsolve
```

Comment:
Why not just call it dogleg?

Reply:
They are all dogleg methods, i.e., they all use the dogleg step to compute the update. The difference between all options is just how the trust region is being adapted.

Reply:
I see, it's a bit of a weird name though. Is there no reference to where it came from? If not then we should credit Spenser Lyon who first implemented what was there.

Reply:
I didn't see anything in the NLsolve docs or code. I just opened an issue. Maybe someone knows more over there. If not, we can also just call it `Lyon`.